components/navigation.html

```diff
@@ -0,0 +1,7 @@
+<nav class="navigation">
+  <a href=""><img src="../icons/DashboardIcon.svg" alt="Dashboard" /></a>
+  <a href=""><img src="../icons/BuddyIcon.svg" alt="Buddy" /></a>
+  <a href=""><img src="../icons/TeamIcon.svg" alt="Team" /></a>
+  <a href=""><img src="../icons/EnergyIcon.svg" alt="Energy" /></a>
+  <a href=""><img src="../icons/JournalIcon.svg" alt="Journal" /></a>
```

Comment:
Is there a reason why the links to the pages are not yet in place?
```suggestion
 You could already insert the links for href-attribute here:
   <a href=""><img src="../icons/DashboardIcon.svg" alt="Dashboard" /></a>
  <a href="../index.html"><img src="../icons/BuddyIcon.svg" alt="Buddy" /></a>
  <a href="../buddies.html"><img src="../icons/TeamIcon.svg" alt="Team" /></a>
  <a href="../energy.html"><img src="../icons/EnergyIcon.svg" alt="Energy" /></a>
  <a href="../journal.html"><img src="../icons/JournalIcon.svg" alt="Journal" /></a>
```